tests/test_chatmemberupdated.py

```diff
@@ -239,3 +241,19 @@ def test_difference_optionals(self, optional_attribute, user, chat):
             chat, user, datetime.datetime.utcnow(), old_chat_member, new_chat_member
         )
         assert chat_member_updated.difference() == {optional_attribute: (old_value, new_value)}
+
+    def test_difference_different_classes(self, user, chat):
+        old_chat_member = ChatMemberOwner(user=user, is_anonymous=False)
+        new_chat_member = ChatMemberBanned(user=user, until_date=datetime.datetime(2021, 1, 1))
+        chat_member_updated = ChatMemberUpdated(
+            chat=chat,
+            from_user=user,
+            date=datetime.datetime.utcnow(),
+            old_chat_member=old_chat_member,
+            new_chat_member=new_chat_member,
+        )
+        diff = chat_member_updated.difference()
+        assert diff.pop('is_anonymous') == (False, None)
+        assert diff.pop('until_date') == (None, datetime.datetime(2021, 1, 1))
+        assert diff.pop('status') == (ChatMember.CREATOR, ChatMember.KICKED)
```

Comment:
the `ChatMember.KICKED` makes me think that we could rename that to `ChatMember.BANNED`? Maybe in #2817?

Reply:
Sounds good, will do :)